app/services/burn_up_calculator.rb

```diff
@@ -6,24 +6,28 @@ def call(project)
 
     calculate_points
 
-    @group_by_day
+    @report_data
   end
 
   private
 
   attr_reader :service_full
 
   def initialize_group_by_day
-    @group_by_day = [
-      { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
-      { name: 'real',  data: service_full.group_by_day },
-      { name: 'ideal', data: service_full.group_by_day.dup }
-    ]
+    @report_data = {
+      group_by_day: [
+        { name: 'today', data: { Date.current => service_full.group_by_day[Date.current] } },
+        { name: 'real',  data: service_full.group_by_day },
+        { name: 'ideal', data: service_full.group_by_day.dup }
```

Comment:
why  dup?

Reply:
honestly I can't say, this was the way `group_by_day` was already declared. I simply made it into a hash and added the `total_backlog_points` to it.